packages/@webex/plugin-meetings/src/metrics/constants.ts

```diff
@@ -3,6 +3,7 @@
 const BEHAVIORAL_METRICS = {
   MEETINGS_REGISTRATION_FAILED: 'js_sdk_meetings_registration_failed',
   MEETINGS_REGISTRATION_SUCCESS: 'js_sdk_meetings_registration_success',
+  MEETINGS_REGISTRATION_STEP: 'meetings_registration_step',
```

Comment:
I left off the "js_sdk_" prefix because I want cantina and SDK steps to go in the same table.